VTEX - Subscriptions API (v3).json

```diff
@@ -2247,13 +2141,85 @@
                         "nullable": true
                     },
                     "frequency": {
-                        "$ref": "#/components/schemas/FrequencyResponse"
+                        "required": [
+                            "interval",
+                            "periodicity"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "periodicity": {
+                                "type": "string",
+                                "nullable": true
+                            },
+                            "interval": {
+                                "type": "integer",
+                                "format": "int32"
+                            }
+                        },
+                        "additionalProperties": false
                     },
                     "validity": {
-                        "$ref": "#/components/schemas/ValidityResponse"
+                        "type": "object",
```

Comment:
add description

Reply:
There were too many small components schemas with only one or two mentions. The whole `PlanResponse` schema has to be checked according to the API, to see if it is up to date. I will do that on the next PRs, in this one I was organizing the schema that was part of another endpoint, just had to add a part in there, so I will ask you to approve the PR with this exception. I will add the `PlanResponse` descriptions in the next PRs.